tools/osbuilder/node-builder/azure-linux/uvm_build.sh

```diff
@@ -55,22 +64,28 @@ if [ "${CONF_PODS}" == "yes" ]; then
 	make KDIR=${UVM_KERNEL_HEADER_DIR}
 	sudo make KDIR=${UVM_KERNEL_HEADER_DIR} KVER=${UVM_KERNEL_VERSION} INSTALL_MOD_PATH=${ROOTFS_PATH} install
 	popd
-
 	echo "Building dm-verity protected image based on rootfs"
 	pushd tools/osbuilder
-	sudo -E PATH=$PATH make DISTRO=cbl-mariner MEASURED_ROOTFS=yes DM_VERITY_FORMAT=kernelinit image
+	sudo -E PATH=${PATH} IMAGE_NAME=${LOCAL_IMAGE_NAME} make DISTRO=cbl-mariner MEASURED_ROOTFS=yes DM_VERITY_FORMAT=kernelinit image
 	popd
 
 	echo "Building IGVM and UVM measurement files"
 	pushd tools/osbuilder
 	sudo chmod o+r root_hash.txt
-	sudo make igvm DISTRO=cbl-mariner IGVM_SVN=${IGVM_SVN}
+	sudo make igvm DISTRO=cbl-mariner IMAGE_NAME=${LOCAL_IMAGE_NAME} IGVM_SVN=${IGVM_SVN} IGVM_DEBUG=${IGVM_DEBUG}
 	popd
 else
 	echo "Building image based on rootfs"
 	pushd tools/osbuilder
-	sudo -E PATH=$PATH make DISTRO=cbl-mariner image
+	sudo -E PATH=$PATH IMAGE_NAME=${LOCAL_IMAGE_NAME} make DISTRO=cbl-mariner image
 	popd
 fi
 
+# We must clean the rootfs build to allow the next build (i.e. a debug image) to be built
+# from a clean state with a separate set of packages. 
+echo "Cleaning rootfs build"
+pushd tools/osbuilder
+sudo -E PATH=$PATH make DISTRO=cbl-mariner clean-rootfs
+popd
```

Comment:
I think we should rather do this when it's needed, ie. at the start of the script - what if you trigger uvm_build.sh and then want to inspect the rootfs content?